JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsApp.kt

```diff
@@ -71,49 +76,55 @@ fun AppDrawer(
     currentScreen: Screen,
     closeDrawer: () -> Unit
 ) {
-    Column(modifier = LayoutSize.Fill) {
-        Spacer(LayoutHeight(24.dp))
-        Row(modifier = LayoutPadding(16.dp)) {
-            VectorImage(
-                id = R.drawable.ic_jetnews_logo,
-                tint = MaterialTheme.colors().primary
-            )
-            Spacer(LayoutWidth(8.dp))
-            VectorImage(
-                id = R.drawable.ic_jetnews_wordmark,
-                tint = MaterialTheme.colors().onSurface
-            )
-        }
-        Divider(color = MaterialTheme.colors().onSurface.copy(alpha = .2f))
+    Column(modifier = Modifier.fillMaxSize()) {
+        Spacer(Modifier.preferredHeight(24.dp))
+        JetNewsLogo(Modifier.padding(16.dp))
+        Divider(color = MaterialTheme.colors.onSurface.copy(alpha = .2f))
         DrawerButton(
             icon = R.drawable.ic_home,
             label = "Home",
-            isSelected = currentScreen == Screen.Home
-        ) {
-            navigateTo(Screen.Home)
-            closeDrawer()
-        }
+            isSelected = currentScreen == Screen.Home,
+            action = {
+                navigateTo(Screen.Home)
+                closeDrawer()
+            }
+        )
 
         DrawerButton(
             icon = R.drawable.ic_interests,
             label = "Interests",
-            isSelected = currentScreen == Screen.Interests
-        ) {
-            navigateTo(Screen.Interests)
-            closeDrawer()
-        }
+            isSelected = currentScreen == Screen.Interests,
+            action = {
+                navigateTo(Screen.Interests)
+                closeDrawer()
+            }
+        )
+    }
+}
+
+@Composable
+private fun JetNewsLogo(modifier: Modifier = Modifier) {
+    Row(modifier = modifier) {
+        Icon(vectorResource(R.drawable.ic_jetnews_logo), tint = MaterialTheme.colors.primary)
```

Comment:
Using `Icon` feels wrong here—it's not an icon. `Image` feels more correct here. If it's just for the simpler `tint` then use the `ColorFilter.tint` ext fun.
```suggestion
        Image(asset = vectorResource(R.drawable.ic_jetnews_logo), colorFilter = ColorFilter.tint(MaterialTheme.colors.primary))
```